src/platform/plugins/shared/usage_collection/server/usage_counters/usage_counters_service.ts

```diff
@@ -163,10 +163,17 @@ export class UsageCountersService {
     counters: UsageCounters.v1.CounterMetric[],
     soRepository: Pick<SavedObjectsRepository, 'incrementCounter'>
   ) {
+    const backoffDelay = (attempt: number) => Math.pow(2, attempt) * 10; // exponential backoff
```

Comment:
Tiny perf improvement: can we define this method as global so that we don't have as many backoffDelay scoped definitions as in-flight storeDate$?

Reply:
> can we define this method as global so that we don't have as many backoffDelay scoped definitions as in-flight storeDate$

By "global", do you mean global to usage counters or global to some other scope?